client/src/components/common/DropDown.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactNode } from 'react';
+import COLORS from 'src/constants/colors';
+import { fixedEntire, flexRow } from 'src/styles/common';
+import styled from 'styled-components';
+
+interface DropDownProps {
+  children: ReactNode;
+  close: () => void;
+}
+
+export default function DropDown({ children, close }: DropDownProps) {
+  const handleClick = (e: React.MouseEvent<HTMLDivElement>) => {
```

Comment:
함수 이름이 좀 더 구체적이면 좋을 것 같아요
```suggestion
  const handleClickContainer = (e: React.MouseEvent<HTMLDivElement>) => {
```

Reply:
네!!